frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeSizeSection.tsx

```diff
@@ -101,6 +100,14 @@ const ServingRuntimeSizeSection: React.FC<ServingRuntimeSizeSectionProps> = ({
               const target = event.currentTarget;
               setData('gpus', parseInt(target.value) || 0);
             }}
+            onBlur={(event: React.FormEvent<HTMLInputElement>) => {
+              const target = event.currentTarget;
+              const gpuInput = parseInt(target.value) || 0;
+              setData(
+                'gpus',
+                gpuInput <= gpuCount ? (gpuInput > gpuCount ? gpuInput : 0) : gpuCount,
```

Comment:
Wut 🤔 

If the value is less than or equal to the total available check to see if the value is greater than the total available? If yes, use the value otherwise zero? The heck is this supposed to be doing?

Reply:
If you're trying to do min/max stuff -- you can call one inside the other.

```
Math.max(min, Math.min(max, value))
```

Reply:
I was watching the discussion, and I was exactly thinking about this line :D 
* we want the minimum of user input vs available gpus
* we want the maximum beween the user input and 0